juju/juju
# boiler plate juju info cmd

## Description of change

Add feature flag for all Charm Hub integration work: "charm-hub"
Add basic code for the new ```juju info``` command part of Charm Hub integration work.  Other pieces such has charm validation and the api need to be worked out.

## QA steps
```console
$ export JUJU_DEV_FEATURE_FLAGS=charm-hub
$ juju bootstrap localhost --build-agent
$ juju info
ERROR expected a charm or bundle name
$ juju info test
ERROR business logic of info command not implemented
```

## Documentation changes

Will be needed for the charm hub integration work for juju 2.9
